### ISSUE TITLE
FossID: Scammer option `fetchSnippetMatchedLines` should be removed

### ISSUE THREAD

**Issue by nnobelis** [oss-review-toolkit/ort#8647]
Due to changes in ORT (https://github.com/oss-review-toolkit/ort/pull/7401), the snippets results are now grouped by source line ranges.
Therefore, missing the matched lines information does not make sense if one wants to work with snippets.

I suggest we remove this option and always assume it's `true`.
This call is expensive though since one such request is made per snippet, but hopefully it will be better when FossID change it's API to an OPENAPI-compliant one.

**Comment by sschuberth** [comment]
> hopefully it will be better when FossID change it's API to an OPENAPI-compliant one.

Has this issue been reported to FossID?

**Comment by nnobelis** [comment]
Yes we opened FOSSIDSC-3099 and they answered that the improvement is on their roadmap.

**Comment by sschuberth** [comment]
Any update here @nnobelis?

**Comment by nnobelis** [comment]
Still no news regarding a new API for snippet :(
